graph/src/data/query/error.rs

```diff
@@ -120,8 +121,8 @@ impl fmt::Display for QueryExecutionError {
             FilterNotSupportedError(value, filter) => {
                 write!(f, "Filter not supported by value `{}`: `{}`", value, filter)
             }
-            UnknownField(_, t, s) => {
-                write!(f, "Type `{}` has no field `{}`", t, s)
+            UnknownField(p, t, s) => {
+                write!(f, "Type `{}` has no field `{}`, at position {}:{}", t, s, p.line, p.column)
```

Comment:
Rather than including the position in the error string, can we add the `UnknownField` variant here?
https://github.com/graphprotocol/graph-node/pull/1339/files#diff-b50193ba4cc86fb70dca7e10af594488R328-R336

That way clients will receive the error as an object, with `line`/`column` fields, as is ideal in GraphQL.

Reply:
Oh I did not notice we have that, everything makes sense now! I've added it there, thanks.

Reply:
Yep, not exactly obvious 😁

Reply:
I still think we should remove the `, at position {}:{}` here because now the position will show up twice in query results w/ this error. 😉

Reply:
Ah of course 🤦‍♂